doc/tools/coqrst/coqdomain.py

```diff
@@ -807,6 +858,7 @@ def setup(app):
     app.connect('doctree-resolved', simplify_source_code_blocks_for_latex)
 
     # Add extra styles
+    app.add_stylesheet("fonts.css")
```

Comment:
It seems that this `fonts.css` file is not in the repository. Is it an omission?